antarest/storage/service.py

```diff
@@ -388,8 +388,12 @@ def _save_study(
 
         if owner:
             study.owner = User(id=owner.impersonator)
-        if group:
-            study.groups = [group]
+            groups = [
```

Comment:
Does it really what we want. I understand that if a user create a study, study belongs to all his groups automatically. It's like if a create a private repo on github, hadar and antares organization can see it.

Reply:
indeed